### PR TITLE
Sprint4 document controllers rg

### PR DESCRIPTION
**Pull request by rebs1203**
Finalized the files upload implementation and edited pet controllers accordingly

### CODE REVIEW

File: src/app.js
Comment (achasek) [inline src/app.js:1]: I like the change to formatting here. The single quotes and the spaces on your imports. Are you using a linter here for this?

File: src/app.js [hunk 1]
```diff
@@ -1,61 +1,61 @@
...
 const path = require('path');
 
 const mainRouter = require('./routes/mainRouter.js');
 
+
 // middleware
 app.use(cors());
 app.use(express.json());
-app.use(express.urlencoded({ extended: false }));
+app.use(express.urlencoded({ extended: true }));
```
Comment (achasek) [inline src/app.js:20]: Is this line right here what fixed the uploading issue with Multer?

File: src/app.js [hunk 2]
```diff
@@ -1,61 +1,61 @@
...
+        const filename = buf.toString('hex') + path.extname(file.originalname);
+        const fileInfo = {
+          filename: filename,
+          bucketName: 'uploads',
+        };
+        resolve(fileInfo);
+      });
     });
-}
+  },
```
Comment (achasek) [inline src/app.js:45]: again, nice formatting changes here!

File: src/controllers/petController.js
Comment (achasek) [inline src/controllers/petController.js:1]: I like the error-handling here. Adding the specifc IDs of the entity throwing the error. Makes it very clear for the user to see what's going on.

File: src/db/connect.js
Comment (achasek) [inline src/db/connect.js:1]: These changes here are more concise. Is there any other reason for this change here?

**Review by achasek — APPROVED** [review]
My only question left and I mentioned this in Slack:

Are we wanting to write unit tests to assert the behavior of the new controller functions that you wrote here?

**Review by shreeup — APPROVED** [review]

**Pull request merged**